subprojects/core/src/main/java/org/gradle/initialization/StartParameterBuildOptions.java

```diff
@@ -564,7 +564,7 @@ public void applyTo(boolean value, StartParameterInternal settings, Origin origi
 
     public static class ConfigurationCacheQuietOption extends BooleanBuildOption<StartParameterInternal> {
 
-        public static final String PROPERTY_NAME = "org.gradle.configuration-cache.quiet";
+        public static final String PROPERTY_NAME = "org.gradle.internal.configuration-cache.quiet";
```

Comment:
Note this is inconsistent with some internal CC flags (i.e. those set only as system properties), as they are named `org.gradle.configuration-cache.internal...`, which from what I saw deviates from our conventions.

Reply:
Why not `org.gradle.configuration-cache.internal.quiet`?

Reply:
I'd keep the flag consistent with the other CC flags. I don't know of other conventions that should take precedence.

Reply:
A search for `propert.*("|')org\.gradle.*internal.*("|')` reveals dozens of Gradle properties and system properties in the code base that use `org.gradle.internal`, so that seems to be the convention.

Also, the C-C "unsafe" properties are also `org.gradle.unsafe.configuration-cache`, instead of `org.gradle.configuration-cache.unsafe`.

I think the two `load-after-store` and `task-execution-access-pre-stable` were unintentional  deviations, which is less important given their transient nature, and the fact they are system properties only. Unfortunately, I contributed to that deviation in the context of implementing support for encryption, but again, also all system properties, meant only to improve testability.

I'd prefer not adding yet another deviation, and a more durable one, given that these `unsafe` properties are actually Gradle properties,  supported in `gradle.properties` files (without requiring a `systemProp` prefix).